diff -Naur enigma2-openvision/main/Makefile.am enigma2-openvision-PersianPrince/main/Makefile.am
--- enigma2-openvision/main/Makefile.am	2019-10-21 03:46:30.425352000 +0330
+++ enigma2-openvision-PersianPrince/main/Makefile.am	2020-01-31 16:27:24.846660642 +0330
@@ -37,7 +37,6 @@
 	@BASE_LIBS@ \
 	@LIBGIF_LIBS@ \
 	@LIBJPEG_LIBS@ \
-	@LIBVUGLES2_LIBS@ \
 	@LIBSDL_LIBS@ \
 	@LIBXINE_LIBS@ \
 	@PTHREAD_LIBS@ \
@@ -51,7 +50,8 @@
 	@AVAHI_LIBS@ \
 	@LIBHIACCEL_LIBS@ \
 	@LIBDL_LIBS@ \
-	-ltuxtxt32bpp
+	-ltuxtxt32bpp \
+	-lv3ddriver @LIBVUGLES2_LIBS@
 
 enigma2_LDFLAGS = -Wl,--export-dynamic
 
