diff -Naur gcc-4.9.4/gcc/config.gcc gcc-4.9.4-PersianPrince/gcc/config.gcc
--- gcc-4.9.4/gcc/config.gcc	2016-03-14 13:33:12.000000000 +0330
+++ gcc-4.9.4-PersianPrince/gcc/config.gcc	2020-02-10 00:11:19.454202177 +0330
@@ -1963,7 +1963,7 @@
 	tm_file="dbxelf.h elfos.h gnu-user.h linux.h linux-android.h glibc-stdint.h ${tm_file} mips/gnu-user.h mips/gnu-user64.h mips/linux64.h mips/linux-common.h"
 	extra_options="${extra_options} linux-android.opt"
 	tmake_file="${tmake_file} mips/t-linux64"
-	tm_defines="${tm_defines} MIPS_ABI_DEFAULT=ABI_N32"
+	tm_defines="${tm_defines} MIPS_ABI_DEFAULT=ABI_64"
 	case ${target} in
 		mips64el-st-linux-gnu)
 			tm_file="${tm_file} mips/st.h"
